buildSrc/subprojects/buildquality/src/main/kotlin/gradlebuild.ci-reporting.gradle.kts

```diff
@@ -92,13 +92,24 @@ fun prepareReportsForCiPublishing(failedTasks: List<Task>, executedTasks: List<T
     val failedTaskCustomReports = failedTasks.flatMap { it.failedTaskGenericHtmlReports() }
     val attachedReports = executedTasks.flatMap { it.attachedReportLocations() }
     val executedTaskCustomReports = failedTasks.flatMap { it.failedTaskCustomReports() }
+    val testDistributionTraceJsons = executedTasks.flatMap { it.findTraceJson() }
```

Comment:
Not sure if it's worth it, but it would be sufficient to do this for `Test` tasks.

Reply:
Thanks, done.